### PR TITLE
DED-564 Update homepage in Docs

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

https://64104e5f9a36997cb106c13b--livechat-public-docs.netlify.app/

# Links

Link your Jira ticket.

- [Jira]()

# Description

Is this what you were thinking about?🤔 I wasn't sure if I could already use the name `Text` as the main company name. I tried to make it look like both places for LiveChat and HelpDesk however as we do not care about HD APIs automatically more info is connected with LiveChat.

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/extending-agent-app/index.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ desc: "You can extend the LiveChat Agent App with widgets, buttons, and other UI
 
 # Introduction
 
-This document presents how you can change and extend the LiveChat App and its interface, so it is personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
+This document presents how you can change and extend the LiveChat and HelpDesk Apps and their interfaces, so they're personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
```
Comment (kamilatokarska) [inline src/pages/extending-agent-app/index.mdx:11]: Noticed that this paragraph was missed by me when adjusting docs to also talk about HD

File: src/pages/index.mdx [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ---
 weight: 0
-title: "LiveChat Platform Docs"
-tagline: "The source of knowledge of the LiveChat Platform"
-desc: "LiveChat is an online customer service software with live support, help desk and web analytics capabilities. Browse the LiveChat Platform documentation to find out more."
+title: "Platform Docs"
+tagline: "The source of knowledge of the Platform Docs"
+desc: "Text Platform is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform Docs documentation to find out more."
```
Comment (kamilatokarska) [inline src/pages/index.mdx:5]: Now I think that changing it to `Platform Docs` would be a better idea🤔 I changed it as the first thing and later forgot about it as it's not visible on the preview.

File: src/pages/index.mdx [hunk 3]
```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
 <SectionBanner
   image
   to={"/extending-agent-app/"}
-  title={"Extending Agent App"}
+  title={"Extending Apps' interfaces"}
```
Comment (kamilatokarska) [inline src/pages/index.mdx:33]: As this section includes SDKs for both products, I decided to change the title here and adjust the description so it mentioned both products.

Comment (ale-kacperczyk) [reply src/pages/index.mdx:33]: I'd drop the `'` and make `apps` singular

File: src/pages/index.mdx [hunk 4]
```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
...
   desc={
-    "With the tools provided by the LiveChat Platform, you can easily extend the Agent App interface. Build apps to add new panels, widgets, and other UI elements."
+    "With Platform Docs's tools, you can easily extend the LiveChat and HelpDesk Apps interfaces. Build apps to add new panels, widgets, and other UI elements."
```
Comment (ale-kacperczyk) [inline src/pages/index.mdx:35]:
```suggestion
    "With Platform tools, you can easily extend the LiveChat and HelpDesk app interfaces. Build apps to add new panels, widgets, and other UI elements."
```

File: src/pages/index.mdx [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+tagline: "The source of knowledge of the Platform Docs"
+desc: "Text Platform is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform Docs documentation to find out more."
 hidden: false
 ---
 
-# About LiveChat Platform
+# About Platform Docs
 
-**LiveChat** is an online customer service software with live support, help desk, and web analytics capabilities. It's used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **LiveChat Platform** is an ecosystem for developers that grew around LiveChat products over the years.
+**Text** is an online customer service software with live support, help desk, and web analytics capabilities. It's used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **Platform Docs** is an ecosystem for developers that grew mainly around two Text products, <a href="https://www.livechat.com/" target="_blank">LiveChat</a> and <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>.
```
Comment (ale-kacperczyk) [inline src/pages/index.mdx:11]: Platform Docs -> Platform (also in some other places)

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: src/pages/index.mdx [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
 
-From the technical perspective the core of LiveChat Platform is the **open messaging protocol**. The Messaging APIs let you integrate any communication channels directly into LiveChat. From the business point of view, we're a team dedicated to build a vivid [LiveChat Marketplace](https://www.livechat.com/marketplace/) of apps and services.
+From the technical perspective the core of Platform Docs is the **open messaging protocol**. The Messaging APIs let you integrate any communication channels directly into LiveChat product. From the business point of view, we're a team dedicated to build vivid [LiveChat Marketplace](https://www.livechat.com/marketplace/) and HelpDesk Marketplace full of apps and services created by <a href="https://developers.livechat.com/" target="_blank">our community of developers</a>.
```
Comment (kamilatokarska) [inline src/pages/index.mdx:13]:
```suggestion
From the technical perspective, the core of the Platform is the **open messaging protocol**. The Messaging APIs let you integrate any communication channels directly into LiveChat product. From the business point of view, we're a team dedicated to build vivid [LiveChat Marketplace](https://www.livechat.com/marketplace/) and HelpDesk Marketplace full of apps and services created by <a href="https://developers.livechat.com/" target="_blank">our community of developers</a>.
```

File: src/pages/index.mdx [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
+tagline: "The source of knowledge of the Platform"
+desc: "Text is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform documentation to find out more."
 hidden: false
 ---
 
-# About LiveChat Platform
+# About Platform
 
-**LiveChat** is an online customer service software with live support, help desk, and web analytics capabilities. It's used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **LiveChat Platform** is an ecosystem for developers that grew around LiveChat products over the years.
+**<a href="https://text.com/" target="_blank">Text</a>** enables entrepreneurs to build technology that provides better customer service apps and helps them reach new audiences. Text products are used by more than <Placeholder id="NUMBER_OF_CUSTOMERS" /> companies in over 150 countries. **Text Platform** is an ecosystem for developers that grew mainly around two Text products, <a href="https://www.livechat.com/" target="_blank">LiveChat</a> and <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>.
```
Comment (kamilatokarska) [inline src/pages/index.mdx:11]: I would add the link to the Text Platform when the website starts

File: src/pages/index.mdx [hunk 8]
```diff
@@ -1,16 +1,16 @@
 ---
 weight: 0
-title: "LiveChat Platform Docs"
-tagline: "The source of knowledge of the LiveChat Platform"
-desc: "LiveChat is an online customer service software with live support, help desk and web analytics capabilities. Browse the LiveChat Platform documentation to find out more."
+title: "Platform Docs"
+tagline: "The source of knowledge of the Platform"
+desc: "Text is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform documentation to find out more."
```
Comment (kamilatokarska) [inline src/pages/index.mdx:5]:
```suggestion
desc: "Text enables entrepreneurs to build technology that provides better customer service apps and helps them reach new audiences. Browse the Platform documentation to find out more."
```

File: src/pages/extending-agent-app/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [inline src/pages/extending-agent-app/index.mdx:11]:
```suggestion
This document presents how you can change and extend the LiveChat and HelpDesk products and their interfaces, so they're personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
```

Comment (ale-kacperczyk) [reply src/pages/extending-agent-app/index.mdx:11]: Let's do it this way so there's a difference between our products and apps built by devs

File: src/pages/index.mdx [hunk 9]
```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
 <SectionBanner
   image
   to={"/extending-agent-app/"}
-  title={"Extending Agent App"}
+  title={"Extending App interface"}
```
Comment (kamilatokarska) [inline src/pages/index.mdx:33]:
```suggestion
  title={"Extending App Interface"}
```

File: src/pages/index.mdx [hunk 10]
```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
...
   desc={
-    "With the tools provided by the LiveChat Platform, you can easily extend the Agent App interface. Build apps to add new panels, widgets, and other UI elements."
+    "With Platform tools, you can easily extend the LiveChat and HelpDesk app interfaces. Build apps to add new panels, widgets, and other UI elements."
```
Comment (kamilatokarska) [inline src/pages/index.mdx:35]:
```suggestion
    "With Platform tools, you can easily extend the LiveChat or HelpDesk app interface. Build apps to add new panels, widgets, and other UI elements."
```

Comment (kamilatokarska) [inline src/pages/index.mdx:35]:
```suggestion
    "With Platform tools, you can easily extend the LiveChat and HelpDesk app interface. Build apps to add new panels, widgets, and other UI elements."
```

File: src/pages/index.mdx [hunk 11]
```diff
@@ -30,9 +30,9 @@ We've just scratched the surface. There is much more for you to discover! Go ahe
 <SectionBanner
   image
   to={"/extending-agent-app/"}
-  title={"Extending Agent App"}
+  title={"Extending App Interface"}
```
Comment (ale-kacperczyk) [inline src/pages/index.mdx:33]:
```suggestion
  title={"Extending App Interfaces"}
```

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request closed without merging**